### PR TITLE
Map2DLocation::is_near_to

### PR DESCRIPTION
**Pull request by randaz81**
Added method Map2DLocation::is_near_to()

### CODE REVIEW

File: doc/release/master/location_is_near_to.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+location_is_near_to {#master}
+-------------------------
+
+## Important Changes
+
+### Libraries
+
+#### dev
+* Added method Map2DLocation::is_similar_to()
```
Comment (drdanz) [inline doc/release/master/location_is_near_to.md:9]:
```suggestion
* Added method Map2DLocation::is_near_to()
```

**Review by drdanz — CHANGES_REQUESTED** [review]
LGTM, just fix the changelog...

**Comment by drdanz** [comment]
@randaz81 `dev::Navigation2DClientTest` test is failing

**Comment by randaz81** [comment]
@drdanz issue fixed, PR rebased and merged

**Pull request merged**